database/migrations/2021_04_01_160256_create_squawk_assignments_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSquawkAssignmentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement(
+            "CREATE TABLE `squawk_assignments` (
```

Comment:
Assuming you cant use the traditional migration format here

Reply:
Unfortunately not due to the string primary key that Laravel aren't willing to address at the moment